sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Microsoft.Online.SharePoint.PowerShell.md

```diff
@@ -558,6 +564,9 @@ Stops further processing of site level trim job that is in-progress.
 ### [Remove-SPOSiteGroup](Remove-SPOSiteGroup.md)
 Removes a SharePoint Online group from a site collection.
 
+### [Remove-SPOSiteManageVersionPolicyJob](Remove-SPOSiteManageVersionPolicyJob.md)
+Stops further processing of site manage version policy job that is in-progress.
```

Comment:
Stops processing of the in-progress manage version policy job for the given site.